models/vista3d/configs/inference.json

```diff
@@ -128,48 +138,57 @@
         "sw_batch_size": "@sw_batch_size",
         "use_point_window": "@use_point_window"
     },
+    "postprocessing_transforms": [
+        {
+            "_target_": "ToDeviced",
+            "keys": "pred",
+            "device": "cpu",
+            "_disabled_": true
```

Comment:
since this is disabled - why don't we remove this transform from post?

Reply:
@heyufan1995 could you please answer this one?

Thanks  
Bin